templates/core/stories.html

```diff
@@ -140,6 +140,29 @@ <h4>IMS Evolve adopts Ubuntu Core to provide IoT-enabled business intelligence i
   <div class="row">
     <h2>Ubuntu Core in robotics</h2>
   </div>
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h4>ARX Fit builds the world’s most effective and efficient fitness machines</h4>
```

Comment:
```suggestion
      <h3 class="p-heading--4">ARX Fit builds the world’s most effective and efficient fitness machines</h3>
```
This breaks heading rank which isn't great for accessibility. From [w3](https://www.w3.org/WAI/tutorials/page-structure/headings/): "Skipping heading ranks can be confusing and should be avoided where possible: Make sure that a `<h2>` is not followed directly by an `<h4>`, for example. It is ok to skip ranks when closing subsections, for instance, a `<h2>` beginning a new section, can follow an `<h4>` as it closes the previous section."

Would you mind applying the same fix throughout?

Reply:
updated, please check once again